### ISSUE TITLE
Triggered by consumption of contaminated food

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9948]
**Preferred term label:**
Triggered by consumption of contaminated food

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0033793

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Brucellosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I would say that this is hard to actually observe -- how do we know that the food is contaminated? This is an inference about what must have happened rather than a phenotypic feature?

**Comment by MickeySegal** [comment]
We have findings that should probably survive but probably combined:

- Consumption of contaminated food
- Undercooked food or unpasteurized dairy contaminated
- Cold deli meat may have become contaminated

We also have:

- Exposure to poor sanitary conditions

How about consolidating as:

- Undercooked food
- Poor sanitary conditions
- Unpasteurized dairy products
- Cold deli meat may have become contaminated over time
